### PR TITLE
upgrade docker version and go

### PR DESCRIPTION
**Pull request by samlawson355**
upgrade docker version and bump go version

docker upgrade to 26.1.3+incompatible broke dktest 🙁

### CODE REVIEW

**Comment by samlawson355** [comment]
@dhui what do you think?

**Review by dhui — COMMENTED** [review]
@samlawson355 thanks for the PR!
We've [migrated](https://github.com/dhui/dktest/blob/master/.github/workflows/go.yml) to GitHub Actions but have kept the old TravisCI and CircleCI configs around to make reverts (or multi-vendor CI runs) easier

**Comment by samlawson355** [comment]
> @samlawson355 thanks for the PR! We've [migrated](https://github.com/dhui/dktest/blob/master/.github/workflows/go.yml) to GitHub Actions but have kept the old TravisCI and CircleCI configs around to make reverts (or multi-vendor CI runs) easier

awesome! thank you @dhui! i have reverted that file 😄

File: rand.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ const (
 )
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano())) // nolint:gosec
```
Comment (dhui) [inline rand.go:14]: I'll change this to use `math/rand/v2`

**Review by dhui — APPROVED** [review]
Sorry for the delay and thanks for your patience @samlawson355 ! I'm going to go ahead and merge your PR and make any necessary fixes/changes

**Pull request merged**